tests/utils/test_contextvars.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import random
+import time
+
+import gevent
+
+
+from sentry_sdk.utils import _is_threading_local_monkey_patched
+
+
+def test_gevent_is_patched():
+    gevent.monkey.patch_all()
```

Comment:
I wonder, does this not affect all of the subsequent tests in the test run?

Reply:
We run each test in its own subprocess because the SDK itself is already so heavy on monkeypatching.

Reply:
Ha, nice.